modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1986,9 +1986,68 @@ export class Wallet implements IWallet {
    * @param params
    */
   async accelerateTransaction(params: AccelerateTransactionOptions = {}): Promise<any> {
+    this.validateAccelerationParams(params);
+
+    params.recipients = [];
+
+    return await this.submitTransaction({
+      ...(await this.prebuildAndSignTransaction(params)),
+      ...BuildParams.encode(params),
+    });
+  }
+
+  private validateAccelerationParams(params: AccelerateTransactionOptions) {
+    if (!params.cpfpTxIds && !params.rbfTxIds) {
+      const error: any = new Error('must pass cpfpTxIds or rbfTxIds');
+      error.code = 'cpfptxids_or_rbftxids_required';
+      throw error;
+    }
+
+    if (params.cpfpTxIds && params.rbfTxIds) {
+      const error: any = new Error('cannot specify both cpfpTxIds and rbfTxIds');
+      error.code = 'cannot_specify_both_cpfp_and_rbf_txids';
+      throw error;
+    }
+
+    if (params.cpfpTxIds) {
+      this.validateCpfpParams(params);
+    }
+
+    if (params.rbfTxIds) {
+      this.validateRbfParams(params);
+    }
+
+    if (params.recipients !== undefined) {
+      if (!Array.isArray(params.recipients) || params.recipients.length !== 0) {
+        throw new Error(`invalid value for 'recipients': must be empty array when set`);
+      }
+    }
+  }
```

Comment:
We dont have codecs in SDK?

Reply:
No, but it would be a great improvement. I saw Otto made on for build params but 99% of it is t.unknown